articles/sendgrid-dotnet-how-to-send-email.md

```diff
@@ -149,7 +149,83 @@ The following examples show how to send a message using the Web API.
             }
         }
     }
+    
+## How to: Send email from ASP .NET Core API using MailHelper class
 
+The below example can be used to send a single email to multiple persons from ASP .NET Core api using the `MailHelper` class of `SendGrid.Helpers.Mail` namespace. For this example we are using ASP .NET Core 1.0. 
```

Comment:
from ASP .NET Core api > from the ASP .NET Core API

Reply:
ok i will modify it . Thanks for the input :)